### DISCUSSION TITLE
Q: Duplicate dependencies? `nsfw@^1.2.9` and `nsfw@^2.1.2`

### DISCUSSION THREAD

**Discussion by kittaakos** [eclipse-theia/theia#9281] [Q&A] [answered]
There are two incompatible `nsfw` dependencies in the dependency chain? Can someone please double-check? Thank you!

```
git rev-parse --short HEAD && yarn why nsfw
3420ef639e
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "nsfw"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "nsfw@2.1.2"
info Reasons this module exists
   - "_project_#@theia#core" depends on it
   - Hoisted from "_project_#@theia#core#nsfw"
info Disk size without dependencies: "2.39MB"
info Disk size with unique dependencies: "3.24MB"
info Disk size with transitive dependencies: "3.24MB"
info Number of shared dependencies: 1
=> Found "@theia/eslint-plugin#nsfw@1.2.9"
info Reasons this module exists
   - "_project_#@theia#eslint-plugin#@theia#core" depends on it
   - Hoisted from "_project_#@theia#eslint-plugin#@theia#core#nsfw"
info Disk size without dependencies: "1.84MB"
info Disk size with unique dependencies: "2.34MB"
info Disk size with transitive dependencies: "2.34MB"
info Number of shared dependencies: 1
✨  Done in 1.05s.
```

From `yarn.lock`:

https://github.com/eclipse-theia/theia/blob/3420ef639e3b9240edd94d207d21d83f98a4e697/yarn.lock#L1148-L1152

https://github.com/eclipse-theia/theia/blob/3420ef639e3b9240edd94d207d21d83f98a4e697/yarn.lock#L1185

https://github.com/eclipse-theia/theia/blob/3420ef639e3b9240edd94d207d21d83f98a4e697/yarn.lock#L9996-L10008

**Comment by vince-fugnitto** [comment, marked as answer]
cc @marechal-p I believe `@theia/eslint-plugin` does not get bumped during releases:

https://github.com/eclipse-theia/theia/blob/3420ef639e3b9240edd94d207d21d83f98a4e697/dev-packages/eslint-plugin/package.json#L1-L9

It pulls an older version of `core` which has an older version of `nsfw`.
I know that `@theia/eslint-plugin` is not published so downstream applications will not have two versions of nsfw.

**Reply by paul-marechal** [reply]
Oh wow. Nice catch, the package will actually be bumped but since I merged it after the release it kept the old ranges.
